src/ConfigurationProcessor.php

```diff
@@ -62,21 +75,41 @@ public function processInstallation(array $configuration)
         }
 
         $nativeModules = $configuration['modules'];
-        $processManager = new ProcessManager(100, 8);
+        $timeout = array_key_exists('timeout', $configuration) ?
+            $configuration['timeout'] :
+            self::WAIT_FOR_PROCESS_STATUS_TIMEOUT
+        ;
 
-        foreach ($nativeModules as $moduleName => $moduleVersion) {
-            $this->io->write(sprintf('<info>Looked into "%s" module (version %s)</info>', $moduleName, $moduleVersion));
+        $processes = array_key_exists('processes', $configuration) ?
+            $configuration['processes'] :
+            self::PARALLEL_PROCESSES
+        ;
 
+        $processManager = new ProcessManager($timeout, $processes);
+        $shouldOverWrite = $this->io->askConfirmation('Do you want to overwrite the existing modules? (y/n)');
+
+        foreach ($nativeModules as $moduleName => $moduleVersion) {
             $package = Package::create($moduleName, $moduleVersion, $this->modulesLocation);
             $modulePath = $this->modulesLocation . $package->getName();
-            if (file_exists($modulePath)) {
+
+            if ($shouldOverWrite && file_exists($modulePath)) {
                 $filesystem = new Filesystem();
                 $filesystem->remove($modulePath);
-                $this->io->write(sprintf('Module "%s" is already installed, deleted.', $package->getName()));
+                $this->io->write(sprintf('Module "%s" will be overwritten.', $package->getName()));
             }
 
-            $command = $this->commandBuilder->getCommandOnPackage(new CreateProject(), $package);
-            $processManager->add($command, $this->modulesLocation);
+            if (!file_exists($modulePath)) {
+                $this->io->write(
+                    sprintf(
+                        '<info>Looked into "%s" module (version %s)</info>',
```

Comment:
What do you mean by "look into"?

Reply:
I mean "look into", I don't understand your question :rofl:

Reply:
Well, the dictionary defines it as "To try to find out information", but I don't think that's the meaning you want to communicate.

How about this?
```suggestion
                        '<info>Installing module "%s" (version %s)</info>',
```

Reply:
already changed